tools/InterfaceGenerator/generator/generators/SmartFactoryBase.py

```diff
@@ -1156,7 +1156,7 @@ def _gen_function_impl(self, function, namespace, class_name):
                     schema_items_decl=self._gen_schema_items_decls(
                         function.params.values()),
                     schema_item_fill=self._gen_schema_items_fill(
-                        function.params.values(), function.since, function.until, function.deprecated, function.removed),
```

Comment:
@SNiukalov Please change history spreading to all RPC parameters together with deprecation. 
SO if the entire RPC is deprecated from <version> and <History> is present - 
All RPC params should be deprecated considering <History>.